c/httpserver.c

```diff
@@ -2472,17 +2472,17 @@ static char *getCookieValue(HttpRequest *request, char *cookieName){
 }
 
 #ifdef __ZOWE_OS_ZOS
-static int isLowerCasePasswordAllowed(){
+int isLowerCasePasswordAllowed(){
```

Comment:
These functions don't seem to belong in `httpserver.h` once they've become public because they have nothing to do with the http server. They need to be moved somewhere more appropriate.

Reply:
What would be a good candidate where to move the functions. Utils, zosaccounts or a new file?

Reply:
No, I don't think so. Besides, it'd add an unnecessary dependency to the http server. Maybe `h/zos.c` since it's based on the structs there. @lchudinov what do you think?

Reply:
`zos.c` and `zos.h` looks good to me.

Reply:
Done.

Reply:
Apologies, I just realized that `httpserver.c` is supposed to be buildable on nix/Win, so this change breaks that, because `zos.c` isn't available there.

Reply:
In that regard, `c/utils.c` may be a better place. Don't forget to add 8 char aliases under `#ifndef __LONGNAME__`, you'll see examples in `h/utils.h`.

Reply:
@ifakhrutdinov maybe we can simply wrap the code blocks that call `isLowerCasePasswordAllowed()` into 
```C
#ifdef __ZOWE_OS_ZOS
//...
 #endif
```

Reply:
@ifakhrutdinov No problem. I moved it to utils.